app.py

```diff
@@ -246,24 +248,24 @@ def discard_inspection(form_id):   # pragma: no cover
 @cross_origin(origins=FRONTEND_URL)
 @swag_from("docs/swagger/search_inspection.yaml")
 def search():   # pragma: no cover
-    return jsonify(error="Not yet implemented!"), HTTPStatus.SERVICE_UNAVAILABLE
-    # try:
-    #     # Database cursor
-    #     cursor = CONN.cursor()
-
-    #     # The search query used to find the label.
-    #     user_id = request.args.get('user_id')
-    #     label_id = request.args.get('label_id')
-    #     query = SearchQuery(user_id=user_id, label_id=label_id)
-
-    #     # TO-DO Send that search query to the datastore
-    #     inspections = inspection.get_all_user_inspection(cursor, query.user_id)
-    #     return jsonify(inspections), HTTPStatus.OK
-    # except Exception as err:
-    #     CONN.rollback()
-    # logger.error(f"Error occurred: {err}")
-    # logger.error("Traceback: " + traceback.format_exc())
-    #     return jsonify(error=str(err)), HTTPStatus.INTERNAL_SERVER_ERROR
+    try:
+        # Database cursor
+        with connect_db(FERTISCAN_DB_URL, FERTISCAN_SCHEMA) as conn:
+            with conn.cursor() as cursor:
+                # The search query used to find the label.
+                username = request.args.get('username')
+                query = SearchQuery(username=username)
```

Comment:
We should keep it as simple as possible for now. The SearchQuery model doesnt seem very useful yet.